app/cdap/components/ColumnInsights/Components/ColumnDataDistribution/index.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core';
+import blue from '@material-ui/core/colors/blue';
+import RenderLabel from 'components/ColumnInsights/Components/common/RenderLabel/index';
+import T from 'i18n-react';
+import React from 'react';
+import BarChart from 'react-bar-chart';
```

Comment:
Please use `vega-lite` for this chart. You can see an example wrapper at https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/VegaLiteChart/index.js

This library hasn't been updated for 6 years and uses a very old version of D3.